pythonrepo/__init__.py

```diff
@@ -16,37 +17,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Python Repo."""
 
-__version__ = """1.1.1"""
-"""This is version 1.1.1 of pythonrepo Template"""
+__module__ = """pythonrepo"""
+"""This is pythonrepo module Template."""
+
+
+__version__ = """1.1.4"""
+"""This is version 1.1.4 of pythonrepo Template"""
 
 
 try:
 	import sys
 	import os
-	if 'pythonrepo' in __file__:
+	if str(__module__) in __file__:
```

Comment:
🤔 better to use dynamic value `__module__` rather than some hardcoded value in this template.